inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongStreamImpl.java

```diff
@@ -67,10 +82,17 @@ public class InlongStreamImpl implements InlongStream {
     /**
      * Constructor of InlongStreamImpl.
      */
-    public InlongStreamImpl(InlongStreamInfo streamInfo, InnerInlongManagerClient managerClient) {
-        this.managerClient = managerClient;
+    public InlongStreamImpl(InlongStreamInfo streamInfo, ClientConfiguration configuration) {
+//        this.managerClient = managerClient;
```

Comment:
Why comment on this line? If it is unnecessary, please remove it.

Reply:
Removed.